### ISSUE TITLE
CSS display:none; is an accessibility problem.

### ISSUE THREAD

**Issue by mgifford** [stylelint/stylelint#3308]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Use of CSS Display None is a common accessibility problem. 

http://a11yproject.com/posts/how-to-hide-content/
http://css-tricks.com/places-its-tempting-to-use-display-none-but-dont/

This little piece of CSS hides the content from a screen reader that may need to know it is there semantically. Flag it and inform the developers.

> Which rule, if any, is this issue related to?

??

> What CSS is needed to reproduce this issue?

e.g.

```css
display: none;
```

> What stylelint configuration is needed to reproduce this issue?

??

> Which version of stylelint are you using?

??

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

??

> What did you expect to happen?

> What actually happened (e.g. what warnings or errors you are getting)?

??

**Comment by evilebottnawi** [comment]
It looks like you just deleted our lovely crafted issue template. It was there for good reasons. Please help us solving your issue by answering the questions asked in this template. I'm closing this. Please either update the issue with the template and reopen, or open a new issue.

**Comment by evilebottnawi** [comment]
@mgifford I don't have time looks on other issue in other repos, we don't have many time on this. Just use template and describe what you want, if you ignore my requests I will ignore you, thanks!

**Comment by jeddy3** [comment]
@mgifford Thank you for the suggestion. You'll be pleased to hear that you can already achieve this in stylelint using the [`declaration-property-value-blacklist`](https://github.com/stylelint/stylelint/tree/master/lib/rules/declaration-property-value-blacklist) (or `*-whitelist`) rule:

```json
{ 
  "rules": {
    "declaration-property-value-blacklist": {
      "display": ["none"]
    }
}
```

Handling this kind of thing at the configuration level is a long standing design decision. It's [covered in the VISION doc as so](https://github.com/stylelint/stylelint/blob/master/VISION.md#limit-language-features):

> Provide rules to limit what language features can be used, including:
> - Enforcing best practices _at the configuration level_ e.g. disallowing the `all` keyword for transitions as it is not performant.

The [limiting rules](https://github.com/stylelint/stylelint/blob/master/docs/user-guide/rules.md#limit-language-features) allow users to target pretty much any part of their CSS code and impose limits on it. 

I'm sorry about the issue template confusion. There's a [pending PR](https://github.com/stylelint/stylelint/pull/3294) to add a template specifically for feature requests, which will hopefully reduce future confusion for us all.

**Comment by ntwb** [comment]
Per the numerous personal attacks @mgifford has made against stylelint team members I'm closing and locking this issue also, @mgifford is no longer welcome here @stylelint.
